<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1717</width>
    <height>957</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>励测客户管理系统</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="tabPosition">
        <enum>QTabWidget::West</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>地区</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QTreeWidget" name="treeWidget">
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>二次</string>
        </attribute>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0" colspan="3">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>filter：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>37</width>
              <height>17</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>key：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSearch">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>search</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>377</width>
              <height>17</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnEditUsers">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>edituser</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnExport">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>export</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnAdd">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDelete">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>del</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QTabWidget" name="tabWidget_2">
          <property name="tabPosition">
           <enum>QTabWidget::South</enum>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>客户数据</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QTableView" name="tableView">
              <attribute name="horizontalHeaderHighlightSections">
               <bool>false</bool>
              </attribute>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_3">
           <attribute name="title">
            <string>二次设备表</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QTableView" name="tableView_2">
              <attribute name="horizontalHeaderHighlightSections">
               <bool>false</bool>
              </attribute>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelTipInfo_2">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>763</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="labelTipInfo">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1717</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <action name="actInsert">
   <property name="text">
    <string>添加</string>
   </property>
   <property name="toolTip">
    <string>插入一行</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actAppend">
   <property name="text">
    <string>添加</string>
   </property>
   <property name="toolTip">
    <string>添加一条数据</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actDelete">
   <property name="text">
    <string>删除</string>
   </property>
   <property name="toolTip">
    <string>删除选中的数据</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actExport">
   <property name="text">
    <string>导出</string>
   </property>
   <property name="toolTip">
    <string>导出表格数据到文本中</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actClose">
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="toolTip">
    <string>退出系统</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
